nservicebus/transports/transactions.md

```diff
@@ -45,11 +45,26 @@ NOTE: MSMQ will escalate to a distributed transaction right away since it doesn'
 snippet: TransportTransactionScope
 
 
-#### Consistency guarantees
+#### Atomicity and consistency guarantees
 
-In this mode handlers will execute inside the `TransactionScope` created by the transport. This means that all the data updates are executed as a whole or rolled back as a whole.
+In this mode handlers will execute inside a `TransactionScope` created by the transport. This means that all the data updates and queue operations are all committed or all rolled back.
 
-Distributed transactions do not guarantee atomicity for an outside observer. For example outgoing messages might be dispatched and processed by some other endpoint before changes to the database get committed. However it is guaranteed that eventually all resources will sync up to reflect the outcome of the transaction.
+**A distributed transaction between the queueing system and the persistent storage guarantees _atomic commits_ but does not guarantee _consistency_.**
+
+Consider a message exchange scenario with a saga that models a simple order lifecycle:
```

Comment:
Consider a message exchange scenario with a saga that models a simple order lifecycle for MSMQ transport and RavenDB persistence: